lib/wsdl.js

```diff
@@ -621,8 +621,10 @@ MessageElement.prototype._createLookupTypeObject = function (nsString, xmlns) {
  * @private
  */
 MessageElement.prototype._getNestedLookupTypeString = function (element) {
+  var ignoredNamespaces = this.options ? this.options.ignoredNamespaces
+      : this.ignoredNamespaces;
   var resolvedType = '^',
```

Comment:
Just an overall thought: wouldn't it be more sufficient if we merge/concat `this.options.ignoredNamespaces` with `this.ignoredNamespaces` on creation-time so we can call `this.ignoredNamespaces` in every case without having to re-assign and double-check it?